etcd/src/jepsen/system/etcd.clj

```diff
@@ -24,17 +24,26 @@
 (def log-file "/var/log/etcd.log")
 
 (defn peer-addr [node]
-  (str (name node) ":7001"))
+  (str (net/ip node) ":2380"))
 
 (defn addr [node]
-  (str (name node) ":4001"))
+  (str (net/ip node) ":2380"))
```

Comment:
Does etcd no longer do name resolution? Should be pretty fast, they're hardcoded in /etc/hosts.